src/Dumplie/Application/Command/Customer/ChangeShippingAddress.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+declare (strict_types = 1);
+
+namespace Dumplie\Application\Command\Customer;
+
+final class ChangeShippingAddress
+{
+    /**
+     * @var string
+     */
+    private $cartId;
+    
+    /**
+     * @var string
+     */
+    private $name;
+    
+    /**
+     * @var string
+     */
+    private $street;
+    
+    /**
+     * @var string
+     */
+    private $postCode;
+    
+    /**
+     * @var string
+     */
+    private $city;
+    
+    /**
+     * @var string
+     */
+    private $countryIso2Code;
+
+    /**
+     * @param string $cartId
+     * @param string $name
+     * @param string $street
+     * @param string $postCode
+     * @param string $city
+     * @param string $countryIso2Code
+     */
+    public function __construct(string $cartId, string $name, string $street, string $postCode, string $city, string $countryIso2Code)
```

Comment:
Why do we use so many variables here? Isn't it better to just use variabile of type Address and get rid of all private variables in this class?